demisto_sdk/commands/common/content/objects/pack_objects/trigger/trigger.py

```diff
@@ -36,5 +36,9 @@ def dump(self, dest_dir: Optional[Union[Path, str]] = None) -> List[Path]:
         created_files: List[Path] = []
         created_files.extend(super().dump(dest_dir=dest_dir))
         new_file_path = created_files[0]
-        shutil.copyfile(new_file_path, new_file_path.parent / self.normalize_file_name())
+        if 'external' in str(new_file_path):
```

Comment:
In case the trigger name contains the word external.
Same for all.
```suggestion
        if new_file_path.name.startswith('external-'):
```